src/Sarif.WorkItems/SarifWorkItemsExtensions.cs

```diff
@@ -75,6 +77,35 @@ public static string CreateWorkItemTitle(this Run run)
                     (locationName == null ? "" : " (in " + locationName + ")");
         }
 
+        public static Dictionary<string, int> FetchToolResultCounts(this SarifLog log)
```

Comment:
>FetchToolResultCounts [](start = 46, length = 21)

Maybe name this "ComputePerToolResultsCount"? @lgolding may also have an idea.

Reply:
@mikefan@microsoft.com your name sounds good. @nickmarston I suggest only using Fetch when retrieving an item from a repository such as a database. Even then I prefer Get.

---
In reply to: [394696836](https://github.com/microsoft/sarif-sdk/pull/1817#discussion_r394696836) [](ancestors = 394696836)